lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -119,8 +121,15 @@ def submit(actions)
 
       # If the response is nil that means we were in a retry loop
       # and aborted since we're shutting down
+      return if bulk_response.nil?
+
       # If it did return and there are no errors we're good as well
-      return if bulk_response.nil? || !bulk_response["errors"]
+      if bulk_response["errors"]
+        @bulk_request_metrics.increment(:partial_successes)
```

Comment:
can I counter suggest with calling it `with_errors` ? because the bulk request actually returns 200, but will contain errors.